spec/adapters/alma_adapter_spec.rb

```diff
@@ -232,16 +232,16 @@
       empty_availability = { "99122426947506421" => {} }
       expect(availability).to eq empty_availability
     end
-
-    it "reports availability (without holding_id) for items in temporary locations" do
+    # Update this test with a different fixture to avoid confusion. We dont use etas anymore.
+    it "reports availability for items in temporary locations" do
       availability = adapter.get_availability_one(id: "9952822483506421")
-      fake_holding = availability["9952822483506421"]["fake_id_1"]
-      expect(fake_holding[:id]).to eq "fake_id_1"
-      expect(fake_holding[:status_label]).to eq "Available"
-      expect(fake_holding[:temp_location]).to eq true
-      expect(fake_holding[:on_reserve]).to eq "N"
+      temporary_holding = availability["9952822483506421"]["online$etasrcp"]
+      expect(temporary_holding[:id]).to eq "online$etasrcp"
+      expect(temporary_holding[:status_label]).to eq "Available"
+      expect(temporary_holding[:temp_location]).to eq true
+      expect(temporary_holding[:on_reserve]).to eq "N"
     end
-
+    # Review this test
```

Comment:
What review is necessary for this test?  Should we ticket it after it is merged?

Reply:
I don't remember :) . I made a note and removed it.